src/Notional.ts

```diff
@@ -88,6 +91,7 @@ export default class Notional extends TransactionBuilder {
       treasury: new Contract(addresses.treasury, TreasuryManagerABI, signer) as TreasuryManager,
       balancerVault: new Contract(addresses.balancerVault, BalancerVaultABI, signer) as BalancerVault,
       balancerPool: new Contract(addresses.balancerPool, BalancerPoolABI, signer) as BalancerPool,
+      exchangeV3: new Contract(addresses.exchangeV3, ExchangeV3ABI, signer) as ExchangeV3,
```

Comment:
Add a null check on this and returned undefined if this is not defined

Reply:
done